### PR TITLE
LowNodeUtilization: use clientset in testing, drop all custom reactors

### PR DESCRIPTION
**Pull request by ingvagabund**
SSIA

### CODE REVIEW

**Comment by seanmalloy** [comment]
/lgtm
/assign @damemi 
/kind cleanup

This is required for #371

**Comment by seanmalloy** [comment]
@damemi this PR will unblock the update to Go 1.15.

File: pkg/descheduler/strategies/lownodeutilization_test.go [hunk 1]
```diff
@@ -803,29 +803,21 @@ func TestWithTaints(t *testing.T) {
 				objs = append(objs, pod)
 			}
 
-			fakePtr := newFake(objs...)
```
Comment (damemi) [inline pkg/descheduler/strategies/lownodeutilization_test.go:806]: Is `newFake` called anywhere else? I don't see it used in this file anymore so we can remove it then right?

**Comment by ingvagabund** [comment]
`newFake` function removed
/hold cancel

**Pull request merged**